src/main/resources/interface-specification.yml

```diff
@@ -4011,6 +4011,166 @@ paths:
               schema:
                 type: integer
               description: Time interval in milliseconds. Allowed requests will be constantly replenished during the interval. At the end of the interval the max allowed requests will be available
+  /purposes/{purposeId}/versions/{versionId}/activate:
+    parameters:
+      - $ref: '#/components/parameters/CorrelationIdHeader'
+      - $ref: '#/components/parameters/IpAddress'
+      - name: purposeId
+        in: path
+        required: true
+        schema:
+          type: string
+          format: uuid
+      - name: versionId
+        in: path
+        required: true
+        schema:
+          type: string
+          format: uuid
+    post:
+      summary: Activate Purpose Version
+      operationId: activatePurposeVersion
+      responses:
+        '200':
+          description: Purpose Version Activated
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/CreatedResource'
```

Comment:
I suppose that here we need to consider [this](https://github.com/pagopa/interop-be-backend-for-frontend/pull/116#discussion_r1117039935).